src/main/java/net/dv8tion/jda/internal/requests/Route.java

```diff
@@ -136,7 +137,7 @@
         public static final Route MODIFY_MEMBER =      new Route(PATCH,  "guilds/{guild_id}/members/{user_id}");
         public static final Route ADD_MEMBER =         new Route(PUT,    "guilds/{guild_id}/members/{user_id}");
         public static final Route GET_MEMBER =         new Route(GET,    "guilds/{guild_id}/members/{user_id}");
-        public static final Route MODIFY_SELF_NICK =   new Route(PATCH,  "guilds/{guild_id}/members/@me/nick");
+        public static final Route MODIFY_SELF =        new Route(PATCH, "guilds/{guild_id}/members/@me");
```

Comment:
A space was deleted before the route path
```suggestion
        public static final Route MODIFY_SELF =        new Route(PATCH,  "guilds/{guild_id}/members/@me");
```